### DISCUSSION TITLE
Is it possible to copy exported BibTeX entries to clipboard?

### DISCUSSION THREAD

**Discussion by ForceBru** [retorquere/zotero-better-bibtex#1973] [Q&A] [answered]
When I right click on a paper, select "Export item..." and set the format as "Better BibLaTeX", this creates a new `.bib` file.

However, I'd like to _add_ this one paper to an already existing `.bib` file. Better BibTeX has "Citation Key Quick Copy", but is there something like "Entire BibTeX entry quick copy"? Right now I have to export the paper to a `.bib` file, open it in a text editor, copy the contents and paste that into the target `.bib` file.

**Comment by retorquere** [comment, marked as answer]
* Go into preferences -> Export -> set Default Format to "Better BibLaTeX", then
* preferences -> advanced -> shortcuts and find the "copy selected items to clipboard" shortcut (cmd-shift-C for me since I'm on a Mac)

When you type the shortcut, the selected item(s) will be on the clipboard in Better BibLaTeX format.
